boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_defconfig

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_NRF53X=y
+CONFIG_SOC_NRF5340_CPUAPP_QKAA=y
+CONFIG_BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP=y
+
+CONFIG_ARM_MPU=y
+
+CONFIG_HW_STACK_PROTECTION=y
+
+CONFIG_ARM_TRUSTZONE_M=y
+
+CONFIG_PINCTRL=y
+CONFIG_GPIO=y
+CONFIG_NFCT_PINS_AS_GPIOS=y
+
+CONFIG_BT=y
+
+# Custom solution used instead of Zephyr I2S driver
+CONFIG_I2S=n
+CONFIG_NRFX_I2S=y
+
+CONFIG_I2C=y
+CONFIG_SPI=y
+CONFIG_ADC=y
+
+# Enable NRFX_CLOCK for ACLK control
+CONFIG_NRFX_CLOCK=y
+
+# Minimal libc is not used. Note that this will impact error codes
+CONFIG_NEWLIB_LIBC=y
+
+# Audio codec LC3 related defines
+# FPU_SHARING enables preservation of the hardware floating point registers
+# across context switches to allow multiple threads to perform concurrent
+# floating point operations.
+CONFIG_FPU=y
+CONFIG_FPU_SHARING=y
+
+# Enable SDHC interface
+CONFIG_DISK_DRIVERS=y
+CONFIG_DISK_DRIVER_SDMMC=y
+
+# Allocate RAM buffer for transferring chunck of data from Flash to SPI
+CONFIG_SPI_NRFX_RAM_BUFFER_SIZE=8
+
+# Config the file system
+CONFIG_FILE_SYSTEM=y
+CONFIG_FAT_FILESYSTEM_ELM=y
+CONFIG_FS_FATFS_LFN=y
+# Using stack for LFN work queue
+CONFIG_FS_FATFS_LFN_MODE_STACK=y
+# exFAT enabled to support longer file names and higher transfer speed
+CONFIG_FS_FATFS_EXFAT=y
+CONFIG_FS_FATFS_MAX_LFN=255
```

Comment:
This should not be in the defconfig file. The board by default should be barebones. I cloned this and started running the tests and they all failed, reason being because every test is trying to pull in the FAT filesystem module, which is a module I excluded.

Reply:
Done